### ISSUE TITLE
Concerning "Colors on file names" setting

### ISSUE THREAD

**Issue by AcarBurak** [rodlie/qtfm#150]
Anything differentiating some files (esp. links in my case) is useful, but those hard-coded colours make text illegible against any background other than very light.

One solution would be to make its text italic, for instance, instead of colourized.

Or you might apply some Qt palette colours (BrightText, ToolTipText etc.) instead of hard-coded ones, so we could set them in [qt5ct](https://sourceforge.net/projects/qt5ct/).

**Comment by AcarBurak** [comment]
By the way, it wouldn't hurt either if links are mentioned as links in file properties!

**Comment by rodlie** [comment]
Yeah, been on my TODO list for a while, but I have been very busy this summer.

**Comment by AcarBurak** [comment]
No problem at all. qtfm is a very functional, no non-sense programme against a sea of stupidity. Thank you.
